x-pack/plugins/observability/public/application/index.tsx

```diff
@@ -62,6 +64,7 @@ export const renderApp = ({
   appMountParameters: AppMountParameters;
   ObservabilityPageTemplate: React.ComponentType<LazyObservabilityPageTemplateProps>;
   kibanaFeatures: KibanaFeature[];
+  usageCollection: UsageCollectionSetup;
```

Comment:
nit: this is declared as an optional dependency
```suggestion
  usageCollection?: UsageCollectionSetup;
```